spec/serializers/provider_serializer_spec.rb

```diff
@@ -84,4 +84,78 @@
 
     it { should eq provider.ucas_preferences.send_application_alerts_before_type_cast }
   end
+
+  describe 'contacts' do
+    describe 'admin contact' do
+      let(:provider) do
+        create :provider,
+               contacts: [admin]
```

Comment:
I will probably just the put 
contacts: [admin, utt, banana, bananas]

and just loop it ...

Reply:
or 

```
	the_types.each do |type|
		describe '#{the_type} contact' do
		  let(:provider) do
			create :provider,
				   contacts: [contact]
		  end

		  let(:contact) { create(:contact, type: the_type) }

		  subject { serialize(provider)['contacts'].find { |c| c[:type] == the_type } }

		  its([:name]) { should eq contact.name }
		  its([:email]) { should eq contact.email }
		  its([:telephone]) { should eq contact.telephone }
		end
	end
```

Reply:
Addressed in the comment below.